### DISCUSSION TITLE
Custom Operators

### DISCUSSION THREAD

**Discussion by vitaly-t** [vitaly-t/iter-ops#199] [Show and tell]
Finally, I've got around to documenting [Custom Operators](https://github.com/vitaly-t/iter-ops/wiki/Custom-Operators) in wiki.

Any thoughts and suggestions are welcome!

---

@RebeccaStevens What's your view on this? (if you're still around :smile:)

**Comment by vitaly-t** [comment]
Created a repo for custom operators - [iter-ops-extras](https://github.com/vitaly-t/iter-ops-extras).

**Comment by vitaly-t** [comment]
@RebeccaStevens Hi Rebecca! Are you still around? I haven't heard from you in a while. There is now a [large repo of custom operators](https://github.com/vitaly-t/iter-ops-extras) that I've been working on. If you are still going to participate in this project, I haven't been able to get around to adding tests for those custom operators, could use some help with that ;)

**Reply by RebeccaStevens** [reply]
Looks good. I should be able to help with the tests and that.

**Reply by vitaly-t** [reply]
Please let me know if you're still going to do that. I'm not looking for anything advanced like in the main repo, since it is not meant to become a distributable package, we need just some very basic tests in place.
